Makefile

```diff
@@ -0,0 +1,113 @@
+# Copyright (C) The Kubernetes Authors.
+# Copyright (C) containerd Authors.
+# Copyright (C) nerdctl Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GO ?= go
+
+# test for go module support
+ifeq ($(shell go help mod >/dev/null 2>&1 && echo true), true)
+export GO_BUILD=GO111MODULE=on $(GO) build -mod=vendor
```

Comment:
This project does not use vendor

Reply:
Thank you.
Well, not use vendor? (I actually "make build" without vendor, and I got the error "containerd not found etc.."
Would you tell me how to build this with some option?

Reply:
`go build` should work with Go 1.15